.github/workflows/end2end.yaml

```diff
@@ -685,11 +683,14 @@ jobs:
           GIT_ACCESS_TOKEN: ${{ secrets.GIT_ACCESS_TOKEN }}
           ZENKO_MONGODB_SHARDED: "true"
           ZENKO_ENABLE_SOSAPI: true
+          TIME_PROGRESSION_FACTOR: 86400
+          TRANSITION_ONE_DAY_EARLIER: false
+          EXPIRE_ONE_DAY_EARLIER: false
       - name: Configure E2E test environment
         run: bash configure-e2e-ctst.sh
         working-directory: ./.github/scripts/end2end
       - name: Run CTST end to end tests
-        run: bash run-e2e-ctst.sh
+        run: bash run-e2e-ctst.sh "" "" "" "" --tags 'not @PRA'
```

Comment:
FYI, the "cleaner" way to avoid this would be to improve the parsing of args in `run-e2e-ctst.sh`:
* parse the args before `--` as before ($1, $2, ...)
* then instead of shifting by 4, shift until the `--`

this would allow using it simply like this: `bash run-e2e-ctst.sh -- --tags 'not @PRA'`

can be improved later though

Reply:
Does this require some sort of parsing or is there a "native" way to do it ?

Reply:
no sure (looping on bash argument is quite native, maybe there could be a smart trick with getopt)... but again : no need to fix right now, we don't really have the time...